packages/react-composites/tests/browser/call/hermetic/LobbyScreen.test.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { expect } from '@playwright/test';
+import {
+  buildUrlWithMockAdapter,
+  defaultMockCallAdapterState,
+  defaultMockRemoteParticipant,
+  defaultMockRemotePSTNParticipant,
+  test
+} from './fixture';
+import { IDS } from '../../common/constants';
+import { dataUiId, isTestProfileStableFlavor, stableScreenshot, waitForSelector } from '../../common/utils';
+
+test.describe('Lobby page tests', async () => {
+  test('Lobby page shows correct strings when joining a group call', async ({ page, serverUrl }) => {
+    const dina = defaultMockRemoteParticipant('Dina');
+    /**
+     * remote participants reflect as this state for a moment before the user connects.
+     *
+     * Dina exists to demonstrate that we don't see the 'Calling' string when joining a group call as a flash.
+     */
+    dina.state = 'Idle';
+    const participants = [dina];
+    const initialState = defaultMockCallAdapterState(participants);
+
+    if (initialState.call) {
+      initialState.call.state = 'Connecting';
+    }
+    initialState.page = 'lobby';
+
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.lobbyScreenTitle));
+
+    expect(await stableScreenshot(page)).toMatchSnapshot('lobbyPage-groupCall.png');
+  });
+
+  test('lobby page shows correct strings when starting a ACS outbound call', async ({ page, serverUrl }) => {
+    test.skip(isTestProfileStableFlavor());
+
+    const joel = defaultMockRemoteParticipant('Joel');
+    joel.state = 'Connecting';
+    const participants = [joel];
+    const initialState = defaultMockCallAdapterState(participants);
+
+    if (initialState.call) {
+      initialState.call.state = 'Connecting';
+    }
+    initialState.page = 'lobby';
+
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.lobbyScreenTitle));
+
+    expect(await stableScreenshot(page)).toMatchSnapshot('lobbyPage-ACScall.png');
```

Comment:
Should this ben renamed to `lobbyPage-outboundCall` (a group call is also an ACS call... and we avoid the use of the acronym ACS).

Reply:
Also, nitty nit on snapshot names. All the snapshots currently-use-dash-and-word. This name is mixing camelCase with that and messing with me :D

Reply:
will do! 😊